alacritty_terminal/src/term/mod.rs

```diff
@@ -1308,10 +1308,9 @@ impl<T: EventListener> ansi::Handler for Term<T> {
     }
 
     #[inline]
-    fn dectest(&mut self) {
-        trace!("Dectesting");
-        let mut template = self.cursor.template;
-        template.c = 'E';
+    fn decaln(&mut self) {
+        trace!("Decalnning");
+        let template = Cell { c: 'E', ..Cell::default() };
 
         self.grid.region_mut(..).each(|c| c.reset(&template));
```

Comment:
Extremely minor complaint, but the placing of the newline here is a bit confusing to me. `template` and the reset code belong together, not the `trace` and `template` statement. So I think in general the newline should be placed above `let template` not below it.

Not really a big deal, but if you plan on changing anything else, this would be a nice minor improvement I think.